.github/codecov.yml

```
@@ -1,3 +1,12 @@
 codecov:
   notify:
     after_n_builds: 4
+  require_ci_to_pass: no
+
+coverage:
+  status: off
+
+comment:
+  layout: "reach, diff, flags, files"
```

```suggestion
  layout: "diff, flags, files"
```

Could you please only remove `reach` from this list?  Other than that it looks good to me.